### ISSUE TITLE
Upload in SDMBench website doesn't work

### ISSUE THREAD

**Issue by Rsugihara01** [zhaofangyuan98/SDMBench#2]
hi,
Thanks for great datasets and the library!

I was trying to do some parameter tuning for Leiden and wanted to evaluate the clustering results.

I succeeded to upload the dataset following your tutorial and pop up said "The task is completed!" as in the tutorials,
but nothing showed up at the bottom of the Upload button and nothing changed in the performance dataframe. 

I am using mac and tried to upload the data with Google chrome and Safari.
should i try to upload it in Windows? or is it just a bug?

**Comment by zhaofangyuan98** [comment]
Hello, thank you for your question. It is possible to accomplish your operation using both a MAC computer and a windows computer. Please try again, after uploading your results in the correct format, the analysis results will be displayed under the upload button, and the Performance button will also display the comparison results.

**Comment by Rsugihara01** [comment]
Thanks for your quick response!

I retried to upload it and there was something weird happened.

I checked format and it was correct.
When I uploaded data to data ID 25 in the MERFISH dataset, a pop-up message appeared saying 'The task is completed!' However, nothing changed on the screen.

I initially uploaded the data on page 3 that showed no result. But changing it from page 3 to page 1, dataID 5 was replaced with dataID 25 and displayed the results.

<img width="1353" alt="Screenshot" src="https://github.com/zhaofangyuan98/SDMBench/assets/67230183/8ee812e6-f327-43bc-9a38-7a0a61f0573a">

I can access the results from that replaced data, but I guess this is not supposed to happen?
